src/scripts/core/axis/linear-scale-axis.js

```diff
@@ -31,11 +31,12 @@
                 this._scale = d3.scale.linear().domain(this._domain);
                 if(this.options.xAxis.min == null && this.options.xAxis.max == null)
                     this._scale.nice();
-                this._setRange();
             } else {
                 this._scale.domain(this._domain);
             }
 
+            this._setRange();
```

Comment:
The y axes respond well -- (can't test log axis b/c I can't figure out how to get Contour to use it)
The y axes did need some updating